sklearn/feature_selection/tests/test_base.py

```diff
@@ -113,3 +116,50 @@ def test_get_support():
     sel.fit(X, y)
     assert_array_equal(support, sel.get_support())
     assert_array_equal(support_inds, sel.get_support(indices=True))
+
+
+def test_featurewise_scorer():
+    X, y = make_classification(random_state=0)
+
+    # spearmanr from scipy.stats
+    sp1 = SelectPercentile(featurewise_scorer(spearmanr), percentile=50)
```

Comment:
I thought it might make the test better and more full proof, I will use just one now as its clear two functions wont matter.